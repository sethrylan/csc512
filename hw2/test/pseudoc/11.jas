.source ./test/pseudoc/11.psd
.class public ./test/pseudoc/11
.super java/lang/Object
;
; standard initializer (calls java.lang.Object's initializer)
;
.method public <init>()V
	aload_0
	invokenonvirtual java/lang/Object/<init>()V
	return
.end method
;
; main()
;
.method public static main([Ljava/lang/String;)V
	push(1138);
	print();
	; done
	return
.end method
