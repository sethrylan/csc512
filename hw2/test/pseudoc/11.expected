.source 11.psd
.class public 11
.super java/lang/Object
;
; standard initializer (calls java.lang.Object's initializer)
;
.method public <init>()V
	aload_0
	invokenonvirtual java/lang/Object/<init>()V
	return
.end method
;
; main() -
;
.method public static main([Ljava/lang/String;)V
	.limit stack 2
	.limit locals 1
	getstatic      java/lang/System/out Ljava/io/PrintStream;
	ldc            1138
	invokevirtual  java/io/PrintStream/println(I)V
	; done
	return
.end method
